doc/check.texi

```diff
@@ -1032,6 +1032,51 @@ respectively.
 Similar to @code{ck_assert_int_*}, but compares two unsigned integer values
 (@code{uintmax_t}) instead.
 
+@item ck_assert_floating_eq
+@itemx ck_assert_floating_ne
+@itemx ck_assert_floating_lt
+@itemx ck_assert_floating_le
+@itemx ck_assert_floating_gt
+@itemx ck_assert_floating_ge
+
+Compares two floating point numbers (@code{float}, @code{double},
+@code{long double}) and displays predefined message with condition and
+values of both input parameters on failure.  The operator used for comparison
+is different for each function and is indicated by the last two letters
+of the function name.  The abbreviations @code{eq}, @code{ne}, @code{lt},
+@code{le}, @code{gt}, and @code{ge} correspond to @code{==}, @code{!=},
+@code{<}, @code{<=}, @code{>}, and @code{>=} respectively. Beware
+using those operators for floating point numbers because of precision
+possible loss on every operation on floating point numbers. For example
+(1/3)*3==1 would return false, because 1/3==1.(3) and cannot be represented
```

Comment:
https://en.wikipedia.org/wiki/Repeating_decimal
It turns out that different countries have different notations of repeating decimal. I had never knew about ellipsis notation. In my country braces are used. Thanks for pointing this.